app/src/main/java/com/sdp13epfl2021/projmag/database/interfaces/UserdataDatabase.kt

```diff
@@ -152,4 +153,26 @@ interface UserdataDatabase {
         onSuccess: () -> Unit,
         onFailure: (Exception) -> Unit
     )
+
+    /**
+     * Uploads the user profile to firebase
```

Comment:
The documentation should not contains information about some implementation (Firebase).
The interface can in fact be used with a non-firebase implementation.